.github/actions/workflow-vars/action.yml

```diff
@@ -5,15 +5,23 @@ inputs:
     description: Don't use changing vars like SHA commit
     # TODO: Change that to real bool when it's supported in GHA.
     default: "false"
+    required: true
   setup-flow:
     description: The chart setup flow either "install" or "upgrade".
     default: "install"
+    required: true
   ingress-hostname-base:
     description: The base of the Ingress hostname.
+    required: true
   platform:
-    description: The deployment cloud platform like GKE or ROSA.
+    description: The deployment cloud platform like GKE or ROSA.  
+    required: true
+  type:
+    description: The platform type platform like openshift or kubernetes.
+    required: true
   identifier-base:
     description: The fixed string in the identifier of the deployment it could be PR number or another specified name.
+    required: true
 # NOTE: This is not an exclusive list, some vars are exported as env var not an output.
```

Comment:
To be honest, I'm slightly confused about the weird mix of stuff being used as outputs and others as environment variables. Very inconsistent.

Reply:
I'm not saying either or is better, it just makes the `chart-test-recipes` so much more difficult to read on what is actually used in the end.